experimental/dds/attributable-map/src/map.ts

```diff
@@ -404,7 +405,10 @@ export class AttributableMapClass
 		localOpMetadata: unknown,
 	): void {
 		if (message.type === MessageType.Operation) {
-			this.kernel.tryProcessMessage(message, local, localOpMetadata);
+			assert(
+				this.kernel.tryProcessMessage(message, local, localOpMetadata),
+				"AttributableMap received Invalid op, possibly from a never version",
```

Comment:
I get that from the older client's perspective it might be technically correct to consider the newer op "invalid" but I think this would make it clearer for people reading the error. Just "invalid" makes my brain think of "known but malformed" or something along those lines. Not super invested in the suggestion, though.
```suggestion
				"AttributableMap received invalid/unrecognized op, possibly from a never version",
```